frontend/src/components/pages/IntakePage.tsx

```diff
@@ -1,38 +1,50 @@
-import React, {useState} from "react";
-import { Button, Text } from "@chakra-ui/react";
+import React, { useState } from "react";
+import { Box, Button, Heading } from "@chakra-ui/react";
+import ReferralForm, { ReferralDetails } from "../intake/ReferralForm";
 
 const Intake = (): React.ReactElement => {
   const [step, setStep] = useState(1);
+  const [referralDetails, setReferralDetails] = useState<ReferralDetails>({
+    referringWorker: "",
+    referringWorkerContact: "",
+    familyName: "",
+    referralDate: "",
+    cpinFileNumber: "",
+    cpinFileType: "",
+    phoneNumber: "",
+  });
 
   const nextStep = () => setStep(step + 1);
 
   const prevStep = () => setStep(step - 1);
 
-  switch(step) {
+  switch (step) {
     case 1:
       return (
-        <div style={{ textAlign: "center", paddingTop: "20px" }}>
-          <Text textStyle="heading">Intake 💨 1 </Text>
+        <Box style={{ textAlign: "center", paddingTop: "20px" }}>
```

Comment:
nit: let's add some spacing at the bottom too
```suggestion
        <Box style={{ textAlign: "center", padding: "20px 0px 20px 0px" }}>
```